buildSrc/src/main/kotlin/kord-module.gradle.kts

```diff
@@ -57,21 +59,31 @@ tasks {
         useJUnitPlatform()
     }
 
-    dokkaHtml.configure {
-        this.outputDirectory.set(project.projectDir.resolve("dokka").resolve("kord"))
+    // configure both dokkaHtml and dokkaHtmlPartial tasks
+    // (dokkaHtmlMultiModule depends on dokkaHtmlPartial, dokkaJar depends on dokkaHtml)
+    withType<AbstractDokkaLeafTask> {
+        // see https://kotlin.github.io/dokka/<dokka version>/user_guide/gradle/usage/#configuration-options
 
-        dokkaSourceSets {
-            configureEach {
-                platform.set(org.jetbrains.dokka.Platform.jvm)
+        failOnWarning.set(true)
 
-                sourceLink {
-                    localDirectory.set(file("src/main/kotlin"))
-                    remoteUrl.set(uri("https://github.com/kordlib/kord/tree/master/${project.name}/src/main/kotlin/").toURL())
+        dokkaSourceSets.configureEach {
 
-                    remoteLineSuffix.set("#L")
-                }
+            jdkVersion.set(Jvm.targetInt)
 
-                jdkVersion.set(8)
+            sourceLink {
+                localDirectory.set(file("src/main/kotlin"))
+                remoteUrl.set(URL("https://github.com/kordlib/kord/blob/${Library.commitHashOrDefault("0.8.x")}/${project.name}/src/main/kotlin/"))
```

Comment:
I don't like that the default branch is hard-coded to `0.8.x` here, it was hard-coded to `master` before which became outdated some time ago.
Any ideas on how this can be avoided?

Reply:
Maybe just use the current branch?

Reply:
But that's also just available in the GitHub CI (where we have the commit hash too), right?

Reply:
Also turns out that using the commit hash breaks source links for https://dokka.kord.dev as the github actions that generate the docs use the commit hash of that repo and not kord's commit hash.